library/oneview_network_set.py

```diff
@@ -162,11 +175,27 @@ def __get_network_uri(self, network_name_or_uri):
                 raise OneViewModuleResourceNotFound(self.MSG_ETHERNET_NETWORK_NOT_FOUND + network_name_or_uri)
 
     def __replace_network_name_by_uri(self, data):
+
         if 'networkUris' in data:
             data['networkUris'] = [self.__get_network_uri(x) for x in data['networkUris']]
         if 'nativeNetworkUri' in data and data['nativeNetworkUri']:
             data['nativeNetworkUri'] = self.__get_network_uri(data['nativeNetworkUri'])
 
+    def __update_connection_template(self, bandwidth):
```

Comment:
Please add a comment mentioning the necessity of the new method(eg: This method will update the bandwidth of a networkSet)

Reply:
Done